commons/build.gradle.kts

```diff
@@ -55,6 +55,10 @@ android {
     composeOptions {
         kotlinCompilerExtensionVersion = libs.versions.kotlinCompilerExtensionVersion.get()
     }
+
+    configurations.all {
+        resolutionStrategy.cacheDynamicVersionsFor(0, TimeUnit.SECONDS)
```

Comment:
does this refresh all the dependencies? or onlye the ones with `isChanging = true`?

Reply:
only those whith the isChanging yes